plugins/GSdx/GSRendererOGL.cpp

```diff
@@ -234,7 +234,14 @@ void GSRendererOGL::EmulateTextureShuffleAndFbmask()
 		// Avious dubious call to m_texture_shuffle on 16 bits games
 		// The pattern is severals column of 8 pixels. A single sprite
 		// smell fishy but a big sprite is wrong.
-		m_texture_shuffle = ((v[1].U - v[0].U) < 256);
+
+		// Tomb Raider Angel of Darkness relies on this behavior to prodce a fog effect.
+		// In this case, the address of the framebuffer 
+		// and texture are the same. 
```

Comment:
While you're at it, you should add the text here to the previous line.

Reply:
Git's finally done being stupid (saying my branch wasn't published) The changes are done.